test/integration/minishift.go

```diff
@@ -105,6 +110,13 @@ func (m *Minishift) executingMinishiftCommand(command string) error {
 }
 
 func (m *Minishift) getOpenShiftUrl() string {
-	cmdOut, _, _ := m.runner.RunCommand("console --url")
+	cmdOut, _, _ := m.runner.RunCommand(ConsoleURL)
 	return strings.TrimRight(cmdOut, "\n")
 }
+
+func (m *Minishift) getRoutingUrl() string {
```

Comment:
_getApplicationRoute_ is probably a better name. That said, how can this even work w/o a parameter. You need to know the route or application name.